src/org-units-selector/OrgUnitsSelector.jsx

```diff
@@ -312,7 +321,20 @@ export default class OrgUnitsSelector extends React.Component {
                         <div style={styles.contentItem}>
                             {roots.map(root => (
                                 <div key={root.path} className={`ou-root ${getClass(root)}`}>
+                                    <FormControlLabel
```

Comment:
This new switch will now always be visible to users. However, in most cases, we will want to keep it as-is, a lot of projects use this component. Proposal: Use two props for `OrgUntiTree` so the caller has more control:

- `showShortName?: boolean` (defaults to false)
- `showNameSetting?: boolean` (defaults to false)

Reply:
Makes total sense. Fixed in https://github.com/EyeSeeTea/d2-ui-components/commit/97676e17